Scoop/Controller/Profile/EditProfileViewController.swift

```diff
@@ -658,19 +658,32 @@ class EditProfileViewController: UIViewController {
     }
 
     @objc private func saveEdit() {
-        let name = nameTextField.getText()?.split(separator: " ")
-        let firstName = String(name?[0] ?? "")
-        let lastName = String(name?[1...].joined(separator: " ") ?? "")
-        let grade = classTextField.getText()
-        let pronouns = pronounsTextField.getText()
-        let phoneNumber = phoneNumTextField.getText()
+        guard let name = nameTextField.getText(), !name.isEmpty,
+              let grade = classTextField.getText(), !grade.isEmpty,
+              let pronouns = pronounsTextField.getText(), !pronouns.isEmpty,
+              let hometown = hometownTextField.getText(), !hometown.isEmpty,
+              let snack = snackTextField.getText(), !snack.isEmpty,
+              let song = songTextField.getText(), !song.isEmpty,
+              let stop = stopTextField.getText(), !stop.isEmpty else {
+                  print("Error: Please complete all fields")
```

Comment:
I am really against having these error statements printed out because it clutters the console and is unnecessary as the UI should tell you if it's invalid.

Reply:
I added the button color change and updated the PR. Richie said that he would change the custom text fields, so displaying the error would be dealt with later. The print statements are just temporary in place of that for now.